corehq/apps/change_feed/consumer/feed.py

```diff
@@ -101,6 +102,9 @@ def get_processed_offsets(self):
     def get_latest_offsets(self):
         return get_multi_topic_offset(self.topics)
 
+    def get_latest_offsets_json(self):
+        return json.loads(kafka_seq_to_str(self.get_latest_offsets()))
```

Comment:
The fact this returns a dict instead of json is some tech debt. the calling function needs a dict that is processed by `json_response`